quodlibet/quodlibet/browsers/playlists/main.py

```diff
@@ -615,7 +615,8 @@ def reordered(self, songs):
             playlist = model[iter][0]
             playlist[:] = songs
         elif songs:
-            playlist = FileBackedPlaylist.from_songs(PLAYLISTS, songs)
+            playlist = FileBackedPlaylist.from_songs(PLAYLISTS, songs,
+                                                     self.__class__.library)
```

Comment:
I'm not sure using `self.__class__.library` is the best way to do this. It may be cleaner to add a library argument to the `reordered` method, but I did not want to have to change the `Browser` base class as well as all its implementations to fix a bug that will probably never happen.

This fixes a bug similar to the one reported in #2564, that is very unlikely to happen. The steps to reproduce are:
 - have some songs in the song list, but no playlist selected (achieved e.g. by selecting an artist in the paned browser and switching to the playlist browser)
 - change the position of a song by drag and dropping it, reordering the song list. This will create a new playlist
 - add a song to this playlist. This song will not be in the playlist anymore after restarting quodlibet.